history.txt

```diff
@@ -1,6 +1,35 @@
 OMERO version history
 =====================
 
+5.2.0 (November 2015)
+----------------------
+
+A full, production-ready release of OMERO 5.2.0; dropping support for Java
+1.6; featuring major upgrading of OMERO.web; re-working of the Java Gateway;
+and introducing new user features and many fixes and performance improvements:
+
+-  improved support for many file formats via Bio-Formats
+-  faster import
+-  Java Web Start has been dropped, it is no longer possible to launch the
+   desktop clients from the web
+-  updated jstree to 3.08 and now using json for all tree loading to
+   substantially improve performance
+-  re-worked display of metadata and annotations in both UI clients
+-  many bugs fixed
+
+Developer updates include:
+
+- the OMERO web framework no longer bundles a copy of the Django package, this 
+  dependency must be installed manually
+- removed FastCGI support, OMERO.web can be deployed using WSGI
+- removed property allowing switching back to old server code to delete or
+  move data. You must migrate to using the new request operations
```

Comment:
You don't actually have to migrate your client code until 5.3. Maybe, "The old request operations, to be removed in 5.3, now always use the new server code."